### PR TITLE
#10539 embed.rst

### PR DESCRIPTION
**Pull request by Konntroll**
Finished editing embed.rst for issue #10539. Let me know if you'd like me to make any changes.

### CODE REVIEW

File: sphinx/source/docs/user_guide/embed.rst [hunk 1]
```diff
@@ -124,7 +123,7 @@ Then the corresponding code on the page might look like:
         .then(function(item) { return Bokeh.embed.embed_item(item) })
     </script>
 
-or with modern syntax:
+Or, with modern syntax, like that:
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/embed.rst:126]: I think this should also read "like this" (a "that" does not make sense to me here)

File: sphinx/source/docs/user_guide/embed.rst [hunk 2]
```diff
@@ -209,11 +208,11 @@ appropriate version replacing ``x.y.z``:
...
             crossorigin="anonymous"></script>
 
-The ``"-widgets"`` files are only necessary if your document includes Bokeh widgets.
-Similarly, the ``"-tables"`` files are only necessary if you are using Bokeh data tables in
-your document.
+The ``"-widgets"`` and ``"-tables"`` files are only necessary if your document
+includes Bokeh widgets and data tables.
 
-For example, to use version ``1.4.0``, including widgets and tables support:
+For example, to use version ``1.4.0`` with support for widgets and tables,
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/embed.rst:214]: let's rev all the versions on this page to `2.2.0`

File: sphinx/source/docs/user_guide/embed.rst [hunk 3]
```diff
@@ -370,33 +371,32 @@ Then inserting the script and div elements into this boilerplate:
         </body>
     </html>
 
-Note that above we have not included the ``"-widgets"`` JS and CSS files, since the
-document does not use Bokeh widgets. If required, the CDN resources are available as HTTPS
-URLs as well.
+Note that this doesn't include JavaScript and CSS files for ``"-widgets"``
+because the document doesn't use any Bokeh widgets.
+
+CDN resources are also available as HTTPS URLs if required.
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/embed.rst:377]: let's make sure we demo only HTTPS URLS throughout this page and then delete this sentence

File: sphinx/source/docs/user_guide/embed.rst [hunk 4]
```diff
@@ -423,44 +423,44 @@ The resulting ``<script>`` tag looks like:
...
-presented to the user. This can be accomplished with the |server_document|
-function, which accepts the URL to a Bokeh server application, and returns
-a script that will embed new sessions from that server any time the script
-is executed.
+If an application is running on a Bokeh server that makes it available at some
+URL, you will typically want to embed the entire application in a web page.
+This way, the page will create a new session and display it to the user every
+time it loads.
+
+You can achieve this with the help of the |server_document| function. This
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/embed.rst:452]:
```suggestion
You can achieve this with the |server_document| function. This
```

**Comment by bryevdv** [comment]
Thanks @Konntroll looks good, just a few comments cc @tcmetzger for a look as well

**Comment by Konntroll** [comment]
Thanks for pointing out these issues, @bryevdv.

With the exception of the example in section 'App sessions' all URLs seem to be in the HTTPS protocol format. URLs in the 'App sessions' example are local, so it's probably fine to leave them as they are in this regard.

I think I changed all the versions to ``2.2.0`` now. However, the latest version is ``2.2.3``. Shouldn't we be using that instead? We do in the chapter on [BokehJS](https://docs.bokeh.org/en/latest/docs/user_guide/bokehjs.html).

**Comment by bryevdv** [comment]
> I think I changed all the versions to 2.2.0 now. However, the latest version is 2.2.3. Shouldn't we be using that instead? We do in the chapter on BokehJS.

until this is automated to actually use the latest version automatically, I think it just needs to be "in the ballpark" of recent, and I supposed I have a slight bias towards x.y.0 versions

File: sphinx/source/docs/user_guide/embed.rst [hunk 5]
```diff
@@ -78,34 +78,33 @@ a div to render into:
 
         item_text = json.dumps(json_item(p, "myplot"))
 
-This output can be used by the ``Bokeh.embed.embed_item`` function on a webpage:
+The ``Bokeh.embed.embed_item`` function can then use this output on a web page:
```
Comment (tcmetzger) [inline sphinx/source/docs/user_guide/embed.rst:81]: You could make this a link to the reference guide:

```suggestion
The :func:`~Bokeh.embed.embed_item` function can then use this output on a web page:
```

**Review by tcmetzger — COMMENTED** [review]
Thank you for your edits!

**Comment by Konntroll** [comment]
Thank you for catching these, @tcmetzger. Think I got everything you mentioned.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks (again) @Konntroll !

**Pull request merged**